docs/events.md

```diff
@@ -0,0 +1,13 @@
+# Understanding Lostromos Events
+
+This document is meant to describe the events that will occur when you use Lostromos with different settings.
+
+## Updates with a Filter
+
+When performing updates and using the `crd.filter` option, we have defined the behavior:
+
+* Old Resource: No Filter, New Resource: No Filter - No-op
```

Comment:
Maybe instead of "No Filter" and "Filter" we can use "Filter annotation exists" and "Filter annotation doesn't exist". Makes it a little more clear.

Reply:
Changed, and moved this to a table instead.